tests/fail/type-too-large.rs

```diff
@@ -1,6 +1,6 @@
 // ignore-32bit
 
 fn main() {
-    let _fat: [u8; (1<<61)+(1<<31)] =
-        [0; (1u64<<61) as usize +(1u64<<31) as usize]; //~ ERROR post-monomorphization error
+    let _fat: [u8; (1 << 61) + (1 << 31)];
+    _fat = [0; (1u64 << 61) as usize + (1u64 << 31) as usize]; //~ ERROR post-monomorphization error
```

Comment:
Rustfmt wanted to do one line for all this. I split into 2 statements to preserve the specificity of the test in checking that the error is triggered on the rhs, not on the binding's pattern or type.